src/test/java/io/jenkins/plugins/postgresql/PostgreSQLConnectionTest.java

```diff
@@ -135,24 +135,23 @@ public void testPostgreSQLConnectionFailureForDelete() throws IOException {
         fail("Expected IOException");
     }
 
-    @Test
-    public void testPostgreSQLConnectionFailureForIsReady() throws IOException {
-        final ToxiproxyContainer.ContainerProxy proxy = toxiproxy.getProxy(postgres, PORT);
-        setConfigurationViaProxy(proxy);
-
-        proxy.setConnectionCut(true);
-
-        try {
-            PostgreSQLFingerprintStorage.get().isReady();
-        } catch (IOException e) {
-            assertThat(e.getMessage(), containsString(CONNECTION_ATTEMPT_FAILED));
-
-            proxy.setConnectionCut(false);
-            assertThat(PostgreSQLFingerprintStorage.get().isReady(), is(false));
-            return;
-        }
-        fail("Expected IOException");
-    }
+//    @Test
```

Comment:
Does it need to be adjusted and restored before merging?

Reply:
Yes https://github.com/jenkinsci/postgresql-fingerprint-storage-plugin/pull/17#discussion_r468621290